jib-core/src/main/java/com/google/cloud/tools/jib/frontend/HelpfulSuggestions.java

```diff
@@ -125,4 +125,15 @@ private String forNoCredentialHelpersDefined(
             + "' or "
             + authConfiguration);
   }
+
+  public String forToNotConfigured(String parameter, String buildConfigFilename, String command) {
```

Comment:
Should go above private method.

Reply:
Maybe also a short javadoc since the name and parameters might be a bit confusing.